cli/install/viz.go

```diff
@@ -2,66 +2,925 @@ package install
 
 // Viz defines the primary Conduit Grafana dashboard, installed via the `conduit install` command.
 const Viz = `{
-      "rows": [
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": "-- Grafana --",
+            "enable": true,
+            "hide": true,
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "type": "dashboard"
+          }
+        ]
+      },
+      "editable": true,
+      "gnetId": null,
+      "graphTooltip": 1,
+      "id": null,
+      "iteration": 1520894444409,
+      "links": [],
+      "panels": [
+        {
+          "content": "<div>\n  <div style=\"position: absolute; top: 0, left: 0\">\n    <a href=\"https://conduit.io\" target=\"_blank\"><img src=\"https://conduit.io/images/conduit-primary-white.svg\" style=\"height: 30px;\"></a>\n  </div>\n  <div style=\"position: absolute; top: 0; right: 0; font-size: 15px\">\n    <a href=\"https://conduit.io\" target=\"_blank\">Conduit</a> is a next-generation ultralight service mesh for Kubernetes.\n    <br>\n    Need help? Visit <a href=\"https://conduit.io\" target=\"_blank\">conduit.io</a>.\n  </div>\n</div>",
+          "gridPos": {
```

Comment:
Is there any way we can wrap this line. I am thinking it might be difficult because it is a template string.

Reply:
Good question. I agree wrapping these would make it a lot more readable. Unfortunately the contents of these files is an exact copy/paste of Grafana's rendering of dashboard JSON. The process of editing a dashboard right now is:
1) Install conduit
2) Edit dashboard in Grafana web UI
3) Export JSON
4) Paste JSON into `viz.go`

We may want to automate this process at some point, which would then allow us to do formatting things like your suggestion.